docs/internal-documentation/maintenance/IssueLabels.md

```diff
@@ -17,38 +18,49 @@ when applicable.
 One label from *Priority* could be attached to an issue, or else the issue is considered
 to have normal priority. A priority label could also be set on a pull request, if applicable.
 
+A label from *Status* can be used if applicable.
+
 ## Labels
 
-Category | Label            | Color  | Used in repository | Description
----------|------------------|--------|--------------------|------------------------------------
-Area     | A-Documentation  | green  | all                | Area: Documentation only.
-Area     | A-TestCase       | green  | "main" or Engine   | Area: Test case specification or implementation of test case.
-Area     | A-Translation    | green  | all                | Area: Documentation of, implementation of or actual translation of text.
-Area     | FA-Backend-conf  | green  | Backend            | Focus Area: Improving backend configuration handling.
-Area     | FA-Basic04       | green  | "main" or Engine   | Focus Area: Implementing Basic04 and adjusting other test cases to it.
-Area     | FA-CDS-CDNSKEY   | green  | "main" or Engine   | Focus Area: Adding test cases for CDS and CDNSKEY.
-Area     | FA-IDN-ASCII     | green  | Backend or GUI     | Focus Area: Moving U-label-to-A-label conversion to GUI and zmtest and away from RPCAPI.
-Area     | FA-MethodNT      | green  | "main" or Engine   | Focus Area: Implementing and migrating to MethodNT for test cases.
-Priority | P-High           | red    | all                | Priority: Issue to be solved before other.
-Status   | S-ReleaseTested  | yellow | all                | Status: The PR has been successfully tested in release testing
-Type     | T-Bug            | red    | all                | Type: Bug in software or error in test case description.
-Type     | T-Feature        | blue   | all                | Type: New feature in software or test case description.
-Type     | T-Question       | blue   | all                | Type: Incoming question.
-Type     | T-TrackingIssue  | blue   | all                | Type: Tracks other issues, PRs or other changes.
-
-## Terms
+Category | Label            | Color  | Used in repository | Scope |Description
+---------|------------------|--------|--------------------|-------|---------------------------------------------------------
+Area     | A-Documentation  | green  | all                | Both  | Area: Documentation only.
+Area     | A-TestCase       | green  | "main" or Engine   | Both  | Area: Test case specification or implementation of test case.
+Area     | A-Translation    | green  | all                | Both  | Area: Documentation of, implementation of or actual translation of text.
+Area     | FA-MethodNT      | green  | "main" or Engine   | Both  | Focus Area: Implementing and migrating to MethodNT for test cases.
```

Comment:
FA-MethodNT -> MethodV2

to MethodNT -> to MethodV2

Reply:
Updated.